server/router/src/test/java/org/infinispan/server/router/utils/HotRodTestingUtil.java

```diff
@@ -0,0 +1,38 @@
+package org.infinispan.server.router.utils;
+
+import org.infinispan.configuration.cache.ConfigurationBuilder;
+import org.infinispan.configuration.global.GlobalConfigurationBuilder;
+import org.infinispan.manager.DefaultCacheManager;
+import org.infinispan.server.hotrod.HotRodServer;
+import org.infinispan.server.hotrod.configuration.HotRodServerConfigurationBuilder;
+
+public enum HotRodTestingUtil {
```

Comment:
Again the same argument... The testing util from `HotRod` module contains almost everything - asserts, startup stuff, topology waiters... I don't want to add more bits to it.